tests/acceptance/data-fixtures/DataFixtures.ts

```diff
@@ -0,0 +1,9 @@
+import { mergeTests } from '@shopware-ag/acceptance-test-suite';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface DataFixtureTypes {
+}
+
+export const test = mergeTests(
+
```

Comment:
empty line needed?

Reply:
I think yes, it's a skeleton and properly needed in the future - at least it does no harm